stable/opsportal/templates/kibana-roles.yaml

```diff
@@ -54,7 +54,11 @@ rules:
 - nonResourceURLs:
   - {{ .Values.kibanaRBAC.path | trimSuffix "/"}}
   - {{ .Values.kibanaRBAC.path | trimSuffix "/" }}/*
+  - {{ .Values.kibanaRBAC.path | trimSuffix "/" }}/api/saved_objects/*
+  - {{ .Values.kibanaRBAC.path | trimSuffix "/" }}/elasticsearch/*
+  - {{ .Values.kibanaRBAC.path | trimSuffix "/" }}/api/timelion/run
   verbs:
   - get
   - head
+  - post
```

Comment:
`view` role with `POST`? Is this like opsportal needing to post to graphql?

cc @jr0d

Reply:
KSphere UI does POST requests to fetch information ("viewing things") too. seems like Kibana UI does a similar thing? not all APIs follow REST standards 🤷‍♂️

Reply:
we also should extract kibana chart from opsportal… AFAIK there is a JIRA somewhere